conda/gperftools/LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
+Copyright (c) 2015-2018, conda-forge
+All rights reserved.
```

Comment:
So we have to include a `conda-forge` license with a Google open source product? Isn't there a Google license we need to use?

Reply:
hmm yeah probably. I'll be sure to figure that out

Reply:
So we're using their recipe you mean, and then overwriting? In that case I could see providing both open source licenses in a single file.